CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Changelog
+
+## Unreleased
+
+### New Features
+
+The highlight of this PR is the new Apply Diff View ([#1202](https://github.com/Nexus-Mods/NexusMods.App/pull/1202)). You can now preview the changes made to disk before applying the Loadout:
+
+![Screenshot of the new Apply Diff View that displays a list of files without different states like "Contents modified" or "Added" as well as file counts for directories and file/directory sizes.](./docs/changelog-assets/3ee7ede1aafade7797185cb7f9f49b2a.webp)
```

Comment:
I would suggest using a structure like what VS Code does with their Release Notes:
![image](https://github.com/Nexus-Mods/NexusMods.App/assets/26797547/0d6c9b54-ae0f-4bcb-8fd5-c4feac5f4f12)

Where there is a bullet list of the highlights and then a more in depth overview.
Then we can add further changes and fixes as a bullet list after that.

Reply:
VS Code pushes multiple medium to big features per release. We don't do that. In this release, we added the apply diff view and three diagnostics. A list of two items followed by those two items being explained doesn't look and feel great.

When we have a release that has multiple medium to big features, we can style the changelog for _that_ release similar to how VS Code does it.